java-checks-test-sources/default/src/main/java/checks/MissingPathVariableAnnotationCheckSample.java

```diff
@@ -187,4 +181,54 @@ public String getPlaceHolder(String id) { // compliant, we don't consider this c
     return "Hello World";
   }
 
+  static class ModelA {
+    @ModelAttribute("user")
+    public String getUser(@PathVariable String id, @PathVariable String name) {
+      return "user";
+    }
+
+    @GetMapping("/{id}/{name}")
+    public String get() { // compliant, @ModelAttribute is always called before @GetMapping to generate the model. In our case model attribute
+      // consume the id and name path variables
+      return "Hello World";
+    }
+
+    @GetMapping("/{id}/{name}/{age}")
+    public String get2(@PathVariable String age) { // compliant
```

Comment:
Should be same line as Noncompliant, and also (although it has no effect on the test itself), its `// Compliant` (with capital C). Happens multiple times in this file.

```suggestion
    @GetMapping("/{id}/{name}/{age}") // Compliant
    public String get2(@PathVariable String age) { 
```